examples/navigation-guards/app.js

```diff
@@ -116,7 +126,18 @@ const GuardMixin = {
   }
 }
 
-const NestedChild = {
+const NestedChild1 = {
```

Comment:
I'm refactoring this to use one single component and use a `key` on the router-view